apisix/plugins/server-info.lua

```diff
@@ -87,36 +80,35 @@ local function get_boot_time()
 end
 
 
-local function uninitialized_server_info()
+local function uninitialize_server_info()
```

Comment:
Rename this function is meaningless to me.

Reply:
Yes, I will discard this modification.